### DISCUSSION TITLE
V1.2 compared to Previous versions for past GHG inventories

### DISCUSSION THREAD

**Discussion by smcnichols94** [USEPA/supply-chain-factors#54] [Q&A]
Hello,

I am wondering whether you recommend going back to previous scope 3 GHG inventories that use the previous versions of the EEIO emissions factors and updating them to use version v1.2 and adjusting backwards for inflation? Or is v1.2 the same method where the previous factors still apply and the base data used in v1.2 is just updated based on what has happened over time but the rest of the method is the same?

Thanks!
Susan

**Comment by bl-young** [comment]
We recommend using the latest version of the factors (currently v1.2). This version includes more current GHG data compared to prior versions. The underlying approach is the same. See the [documentation](https://pasteur.epa.gov/uploads/10.23719/1528686/documents/Aboutthe2019v1.2SupplyChainGHGEmissionFactors.pdf) for a complete description of what has changed.
